polyfill-crypto/index.js

```diff
@@ -1,3 +1,10 @@
+// This code modifies `react-native-webview-crypto` package to handle
+// `onContentProcessDidTerminate` by remounting the `WebView` and re-initializing
+// the worker.
+
+// TODO: replace this with webview-crypto/react-native-webview-crypto package once
+// https://github.com/webview-crypto/react-native-webview-crypto/pull/30 is merged
+
 const React = require("react");
```

Comment:
Ignore the diffs below, I just wanted the code to match how it looks in the react-native-webview-crypto PR